pyworkflow/gui/tree.py

```diff
@@ -629,4 +629,25 @@ def getText(self, obj):
         return obj.pluginName + '-' + obj.templateName
 
     def getValues(self, obj):
-        return (obj.description, )
\ No newline at end of file
+        return (obj.description, )
+
+# From: https://stackoverflow.com/questions/1966929/tk-treeview-column-sort
+def treeview_sort_column(tv, col, reverse, casting=str):
```

Comment:
We use camelCase convention for methods, please keep it consistent

Reply:
Moreover, why this method is not implemented as part of the class pwgui.tree.Tree (that is a subclass of Tkinter.Treeview) and it is more OO?

Reply:
Good points, I missed boths. Doing them. Thanks